static/src/javascripts/projects/commercial/modules/comscore.js

```diff
@@ -54,15 +53,13 @@ const initOnConsent = (state: boolean | null) => {
 
 export const init = (): Promise<void> => {
     if (commercialFeatures.comscore) {
-        if (shouldUseSourcepointCmp()) {
-            onConsentChange(state => {
-                if (state.tcfv2) {
-                    initOnConsent(state.tcfv2.vendorConsents[SOURCEPOINT_ID]);
-                }
-            });
-        } else {
-            oldCmp.onGuConsentNotification('performance', initOnConsent);
-        }
+        onConsentChange(state => {
+            if (state.tcfv2) {
+                initOnConsent(state.tcfv2.vendorConsents[SOURCEPOINT_ID]);
+            } else if (state.ccpa) {
+                initOnConsent(!state.ccpa.doNotSell);
+            }
```

Comment:
I've introduced a check agains the CCPA framework, which was previously missing.